app/.gitignore

```diff
@@ -1,4 +1,2 @@
-/build
-
 # wakeword models
-*.lite
```

Comment:
This was wrong.
Btw, what's the point to not add it here ? It's make the app not work out of the box ?

![image](https://user-images.githubusercontent.com/3314607/115985840-fe576e00-a5ad-11eb-8a0e-2f70f4c5b318.png)

Reply:
This was the extension we used to use for TFLite models.

As to why they're not included by default, our base wake word models get updated occasionally, and we'd prefer people running across the sample projects to retrieve the latest versions to get the best performance. They don't automatically download both because this is meant to be a barebones sample, and pre-downloading the models ensures it's ready to use as quickly after launch as possible.